### PR TITLE
chore(datasets): auto-mount fastai public dataset

### PR DESCRIPTION
**Pull request by cwetherill-ps**
Verified that fastai dataset is auto-mounted:
image

and that the path is being correctly set and referenced from the config file:
image

### CODE REVIEW

File: fastai-fastbook/run.sh [hunk 1]
```diff
@@ -4,4 +4,11 @@ export SHELL=/bin/bash
 mkdir /storage/data
 rm -rf /storage/lost+found
 ln -s /datasets/fastai/* /storage/data/
+
+mkdir /notebooks/.gradient
+echo "integrations:
+  fastai:
+    type: dataset
+    ref: paperspace/dsr4qbcrcg662pf:latest" > /notebooks/.gradient/settings.yaml
```
Comment (bbatha) [inline fastai-fastbook/run.sh:12]: Is this not in the `gradient` namespace?

**Review by bbatha — APPROVED** [review]

File: fastai-fastbook/run.sh (hunk 1 above)
Comment (cwetherill-ps) [reply fastai-fastbook/run.sh:12]: 🤷 I was just replicating the yaml that's auto-generated when you manually mount the public dataset into a notebook. Would we expect them to produce the same result, or is there a reason for the different namespacing?

Comment (bbatha) [reply fastai-fastbook/run.sh:12]: Not sure @lamroger

Comment (lamroger) [reply fastai-fastbook/run.sh:12]: `gradient` is better since thats how users reference
https://docs.paperspace.com/gradient/data/#public-datasets

Comment (lamroger) [reply fastai-fastbook/run.sh:12]: both should work

Comment (cwetherill-ps) [reply fastai-fastbook/run.sh:12]: should the codepath that generates settings.yaml when you click 'mount' from the UI follow the same logic then? Currently it doesn't. Doesn't change behavior in the notebook, just could be confusing if someone is looking at their settings yaml.

**Pull request merged**